app/controllers/forms_controller.rb

```diff
@@ -117,8 +119,36 @@ def restore
     end
   end
 
+  def actions_toolbar
+    render json: {
+      actions:
+        Toolbars::FormsService.new(
+          current_user,
+          form_ids: JSON.parse(params[:items]).map { |i| i['id'] }
```

Comment:
Hm, why not send just an array of ids to the backend? We only need a more complex structure in the case of Projects+Folders, here just use IDs.